src/renderer/screens/exchange/Swap2/Form/FormSummary/SectionFees.js

```diff
@@ -12,7 +12,37 @@ import sendAmountByFamily from "~/renderer/generated/SendAmountFields";
 import type { SwapTransactionType } from "~/renderer/screens/exchange/Swap2/utils/shared/useSwapTransaction";
 import { rateSelector } from "~/renderer/actions/swap";
 import FormattedVal from "~/renderer/components/FormattedVal";
+import Box from "~/renderer/components/Box";
 import Text from "~/renderer/components/Text";
+import TachometerHigh from "~/renderer/icons/TachometerHigh";
+import TachometerLow from "~/renderer/icons/TachometerLow";
+import TachometerMedium from "~/renderer/icons/TachometerMedium";
+import styled from "styled-components";
+
+type Strategies = "slow" | "medium" | "fast" | "advanced";
```

Comment:
Why are we not using the `SelectFeeStrategy` component instead of redoing it here? Seems like we are reinventing the component

Reply:
I don't get it. The `SelectFeesStrategy` implements all the logic needed to select a fees strategy. Here I just need a logic-less component that displays one of the tachometer icons based on the strategy passed as a props. This component is just informative. Am I missing something? 🤔

Reply:
Note: I wanted to reuse the current `feeStrategy` type to not have to reimplement mine but
- the type isn't isolated and only lives in TransactionCommon/TransactionCommonRaw types
- the advanced strategy is missing from the string literal types
- the type passing between LLD/LLM and LLC is broken since we migrate LLC to typescript


That's why I created a new type which is a string literal type that includes the "advanced" value. I can update the current type located to LLC if needed.

Reply:
🤷🏼‍♂️ I trust you